tests/libYARP_OS/NodeTest.cpp

```diff
@@ -257,6 +265,11 @@ void NodeTest::typePropTest() {
 
 void NodeTest::runTests() {
     NetworkBase::setLocalMode(true);
+
+#ifdef BROKEN_TEST
```

Comment:
Fine for me, but what is the purpose of this BROKEN_TEST macro? i.e. why this test is not always executed?